### DISCUSSION TITLE
Move acronyms list to the bottom of the html document

### DISCUSSION THREAD

**Discussion by matog** [rchaput/acronyms#18] [Q&A] [answered]
Hi! Great extension!
As the title says, is it possible to move the acronyms list to the bottom of the html document?

Is it possible to change the "Acronyms list" title to another name? I'm writting an spanish document, so it would be great if I could translate the title.

**Comment by rchaput** [comment, marked as answer]
Hi! Thank you!

- You can place the acronyms list at the very end of the document by using the `insert_loa: end` option (as documented on https://rchaput.github.io/acronyms/articles/options.html#insert_loa). Alternatively, if you want to choose the location yourself, you can also disable the automatic insertion with `insert_loa: false` and then using `\printacronyms` where you want the acronyms list to appear.

- The list title can be changed by using the `loa_title` option (as documented on https://rchaput.github.io/acronyms/articles/options.html#loa_title), for example: `loa_title: "Lista de acrónimos"` (my Spanish is rusty, sorry if the translation is not correct!).

Now that you mention it, I believe it would be a great addition to be able to automatically detect the document language and translate the list title. I will see what I can do.

**Reply by matog** [reply]
Great!!!!
Sorry, I wasn't aware of the documentation site.
I'm going to read it all!
